ci-jobs/scripts/start-emulator.sh

```diff
@@ -16,8 +16,31 @@ echo "Starting emulator"
 
 # Start emulator in background
 nohup $ANDROID_HOME/emulator/emulator -avd $ANDROID_AVD -no-snapshot > /dev/null 2>&1 &
-$ANDROID_HOME/platform-tools/adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed | tr -d '\r') ]]; do sleep 1; done; input keyevent 82'
 
-$ANDROID_HOME/platform-tools/adb devices
+$ANDROID_HOME/platform-tools/adb wait-for-device get-serialno
+secondsStarted=`date +%s`
+TIMEOUT=360
+while [[ $(( `date +%s` - $secondsStarted )) -lt $TIMEOUT ]]; do
+  # Fail fast if Emulator process crashed
+  pgrep -nf avd || exit 1
+
+  processList=`adb shell ps`
+  if [[ "$processList" =~ "com.android.systemui" ]]; then
+    echo "System UI process is running. Checking IME services availability"
+    $ANDROID_HOME/platform-tools/adb shell ime list && break
+  fi
+  sleep 5
+  secondsElapsed=$(( `date +%s` - $secondsStarted ))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=5515858)